app/assets/stylesheets/partials/_event_box.scss

```diff
@@ -0,0 +1,51 @@
+.event-box {
+    width: 350px;
```

Comment:
Other than the scaling factor listed below, the decision was relatively arbitrary since we will be removing the fixed with once the event box is inside a grid.

Reply:
Alright yeah let's kick the can down the road on this one to the grid implementation then. Also was thinking between this and the carousel I'm doing whether we should be defining these widths in rem units? Should be easy to refactor when the time comes either way.

Reply:
I recommend layout dimensions use percentages or vw/vh since they are based on containers, while rems are in reference to content dimensions.